visualstimulation/utils.py

```diff
@@ -1,12 +1,42 @@
-import os
 import quantities as pq
-import exdir
 import numpy as np
 import neo
+import os
+
+
+def generate_gradiently_weighed_data(data, weight_start=1, weight_end=0):
+    """
+    Creates weighed data using gradients from weight_start to weight_end.
+    Example:
+    In:
```

Comment:
Great that you have included an example, but to keep the style consistent, please use this style: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html (`def example_generator(n)`)